packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.jsx

```diff
@@ -170,6 +171,22 @@ const Amount = ({
                     />
                 </React.Fragment>
             )}
+            {contract_type === 'vanilla' && (
+                <section className='trade-container__stake-field'>
+                    <div className='trade-container__stake-field--min'>
+                        <Text size='xxxs'>Min. Stake</Text>
+                        <Text size='xxs'>
+                            {stake_boundary.min_stake} {currency}
+                        </Text>
+                    </div>
+                    <div className='trade-container__stake-field--max'>
+                        <Text size='xxxs'>Max. Stake</Text>
+                        <Text size='xxs'>
+                            {stake_boundary.max_stake} {currency}
+                        </Text>
+                    </div>
+                </section>
+            )}
```

Comment:
Localization here as well

Reply:
Not needed as the values are numeral and currency unit